src/main/java/com/ioet/bpm/peopletime/domain/TimeTemplate.java

```diff
@@ -0,0 +1,42 @@
+package com.ioet.bpm.peopletime.domain;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAutoGeneratedKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@DynamoDBTable(tableName = "time_template")
+public class TimeTemplate {
```

Comment:
Same here:

The idea is to prepend a prefix for the tables. For example for the service `people` if we are creating the table `time`. The table full name should be:

people_time

Please add the suffix for this table